files/www/cgi-bin/status

```diff
@@ -260,7 +260,7 @@ if($config eq "mesh" and ! $wifi_disable )
     push @col2, $str;
 }
 
-push @col2, "<th align=right><nobr>firmware version</nobr><br>configuration</th><td><nobr>" . `cat /etc/mesh-release` . "</nobr><br>$config</td>";
```

Comment:
It is still used to detect when the node is not yet configured as pulled from an 'nvram' call, and displays the setup instructions accordingly.  While this value could be removed, and figure out what else to key off of,  it would be a super-set change affecting several files.  suggest we do separately or pull out with UI re-write.